src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "name": "Google BigQuery API",
+    "documentation": "https://cloud.google.com/bigquery/docs/reference/rest/v2/",
```

Comment:
Users can go to the product docs, which we link to, if they want to find out about the rest api.  This is a Java library importer wizard.  Let's focus on the information pertinent to that use-case